src/block/cards__container.scss

```diff
@@ -0,0 +1,15 @@
+.cards-block {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  max-width: 1024px;
```

Comment:
checklist 13 (dynamic max-width calculation)

Reply:
agree, it's better to calculate 1024 value instead of hardcoding